StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
+
+    [string]
+    $OutputFolder = "c:\temp\",
+
+    [string]
+    $OutputFile = "binskim",
```

Comment:
>$OutputFile [](http://example.com/codeflow?start=4&length=11)

It'd be nice if this was an identifier encoded in something else, like 'BinSkimHASHESRUN'